traffic-manager-application-gateway-demo-setup/azuredeployappgw.json

```diff
@@ -0,0 +1,342 @@
+﻿{
```

Comment:
this should be in a subfolder - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#nested-templates